src/Arcus.Observability.Telemetry.AspNetCore/Extensions/ILoggerExtensions.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Arcus.Observability.Telemetry.Core;
+using GuardNet;
+using Microsoft.AspNetCore.Http;
+
+// ReSharper disable once CheckNamespace
+namespace Microsoft.Extensions.Logging
+{
+    public static class ILoggerExtensions
+    {
+        private const string RequestFormat =
```

Comment:
What if we move this to a constant that everyone can use in case they want?

That way you can also re-use it here

Reply:
Could be useful, yes. Mostly useful for us I think but yeah, I'll change that. Thanks.

Reply:
Resolving this, as I've made the `RequestFormat` `public`.